submissions/YuliiaHordiichuk/friends-app/app.js

```diff
@@ -0,0 +1,130 @@
+const randomUsersUrl = 'https://randomuser.me/api/?results=36&inc=gender,name,email,phone,picture,dob';  
+
+let users; 
+let searchBy = ''; 
+let sortBy = ''; 
+let filterBy = 'all'; 
+ 
+function getData() {
+    fetch(randomUsersUrl)
+        .then(answer => answer.json())
+        .then(( { results } ) => {
+            users = results;
+            createCards(results); 
+        })
```

Comment:
This should be done outside of the function which gets data.

Pls, read about single responsibility. Right now function named `getData` will also create cards.